README.md

```diff
@@ -31,12 +40,15 @@ After you update the manifest updater and test the changes, you can build an RPM
    $ sudo dnf install subversion
    $ sudo dnf install rpm-build
    ```  
+2. Clone this repository.
```

Comment:
@MichalMaler - how would you feel about adding a source section showing the clone command?

Does this mean we should also make it a prerequisite that a user has registered their SSH keys in GitHub?

Reply:
Good idea! Adding